dev-util/emscripten/emscripten-1.34.11.ebuild

```diff
@@ -18,22 +18,28 @@ LICENSE="UoI-NCSA"
 REQUIRED_USE="${PYTHON_REQUIRED_USE}"
 
 DEPEND="${PYTHON_DEPS}
-	=dev-util/emscripten-fastcomp-1.34.11
+	=dev-util/emscripten-fastcomp-${PV}
```

Comment:
you might want to use ```~``` so that you can revbump ```dev-util/emscripten-fastcomp```. See https://devmanual.gentoo.org/general-concepts/dependencies/#version-specifiers